Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -37,8 +37,17 @@ public protocol PurchasesType: AnyObject {
      * will turn up every time the app is opened.
      * More information on finishing transactions manually [is available here](https://rev.cat/finish-transactions).
      */
+    @available(*, deprecated, message: "Use purchasesAreCompletedBy instead.")
     var finishTransactions: Bool { get set }
 
+    /** Whether transactions should be finished automatically. `.revenueCat` by default.
+     * - Warning: Setting this value to `.myApp` will prevent the SDK from finishing transactions.
+     * In this case, you *must* finish transactions in your app, otherwise they will remain in the queue and
+     * will turn up every time the app is opened.
+     * More information on finishing transactions manually [is available here](https://rev.cat/finish-transactions).
+     */
```

Comment:
For clarity, should we maybe word it like "Purchases will be made and transactions will be finished by RevenueCat"? 🤔

Reply:
Yes!